rfs_test/__init__.py

```diff
@@ -36,8 +37,9 @@ def run(sut):
         run_single([TEST_protocol_details, TEST_datamodel_schema], sut, log)
     else:
         # Run all assertions
-        TEST_protocol_details.run(sut, log)
-        TEST_datamodel_schema.run(sut, log)
+        #TEST_protocol_details.run(sut, log)
+        #TEST_datamodel_schema.run(sut, log)
```

Comment:
It would be good to debug why these need to be commented out; I know others are running the tool on systems and have the above tests run.